pages/linux/tracepath.md

```diff
@@ -0,0 +1,28 @@
+# tracepath
+
+> Trace the path to a network host discovering MTU along this path.
+> More information: <https://manned.org/tracepath>.
+
+- Trace the path to a host:
+
+`tracepath {{host}}`
+
+- Print numerical IP addresses only:
+
+`tracepath -n {{host}}`
+
+- Print both hostnames and numerical IP addresses:
+
+`tracepath -b {{host}}`
+
+- Specify a maximum TTL (number of hops):
+
+`tracepath -m {{max_hops}} {{host}}`
+
+- Specify the initial destination port, preferably 33434:
+
+`tracepath -p {{destination_port}} {{host}} `
```

Comment:
Why not just put the port in the example?
```suggestion
- Specify the initial destination port:

`tracepath -p {{destination_port}} {{33434}} `
```

Reply:
Note that the man page with the hint I linked above is for `traceroute` command, not the `tracepath` discussed here (hopefully everyone noticed). These two commands are only loosely related and from my research there is no comprehensive, easily accessible explanation around the internet about this topic.

I believe there are three scenarios:
1. without port - not useful, not sure what it even does tbh
2. with arbitrary port - useful for testing firewall settings
3. with port 33434 - useful for finding path (what this command should do)

Maybe we could transform it in a way that case 1. would not be even mentioned, since it is not very useful as is (at least on my machine).
Both, case 2. and 3. should then be documented as two separate use cases. This would provide me as, a semi-advanced user of this command, the most usage details - exactly what I look for when I use tldr. Not entirely sure about how the beginner user would benefit. 

What do you think?

Reply:
I agree.  Remove this example and document cases 2 & 3. 👍🏻

Reply:
@bl-ue I considered the "comments" as you guided me, but the new comment push is much better aligned with my workflow. I will keep pushing commits, unless someone objects. See https://github.com/tldr-pages/tldr/pull/5744/commits/714d8d6d7be51d80b4b4db331ffb43c54f82c748